master/buildbot/status/web/base.py

```diff
@@ -456,6 +457,7 @@ def getContext(self, request):
         return dict(title_url = status.getTitleURL(),
                     title = status.getTitle(),
                     stylesheet = rootpath + 'default.css',
+                    custom_styles = os.path.isfile(custom_css),
```

Comment:
Should we cache this result as this will check the disk on every single request from katana